hypervisor/arch/x86/vtd.c

```diff
@@ -1055,13 +1058,17 @@ static int32_t add_iommu_device(struct iommu_domain *domain, uint16_t segment, u
 	uint64_t hi_64;
 	uint64_t lo_64 = 0UL;
 	int32_t ret = 0;
+	union pci_bdf sid;
```

Comment:
what does sid mean? add a comment?

Reply:
> what does sid mean? add a comment?

It is source id. Will add a comment.

Reply:
@dongyaozu A new version is pushed with the comment added. Could you please help review again?